packages/fxa-settings/src/components/ThirdPartyAuth/en.ftl

```diff
@@ -4,7 +4,7 @@
 # This appears when a user has the option to authenticate via third party accounts in addition to their Firefox account. 
 # Firefox account login appears on top, and third party options appear on bottom. 
 # This string appears as a separation between the two, in the following order: "Enter your password" "Or"(this string) "Continue with Google"(continue-with-google-button) / "Continue with Apple"(continue-with-apple-button)
-third-party-auth-options-or = Or
+third-party-auth-options-or-2 = or
```

Comment:
This is mostly an English stylistic change, so can be changed without updating the ID. 
```suggestion
third-party-auth-options-or = or
```